config/deploy.js

```diff
@@ -129,6 +133,42 @@ function deployBuild(url, folder) {
     }
 }
 
+function buildAssetHistory(url) {
+    shell.exec('git clone ' + url + ' current_build');
+    let now = (new Date().getTime()) / 1000;
+    let oldHistoryPath = path.resolve(__dirname, 'current_build/history.json');
+    let newHistoryPath = path.resolve(__dirname, 'history.json');
+    let oldAssetsPath = path.resolve(__dirname, 'current_build/bundles');
+    let newAssetsPath = path.resolve(__dirname, 'bundles');
+
+    // Parse old history file if it exists
+    let history = {};
+    try {
+        history = JSON.parse(fs.readFileSync(oldHistoryPath).toString());
+    } catch (e) {}
+
+    // Add new asset files to history, with current timestamp
+    let newAssets = fs.readdirSync(newAssetsPath);
```

Comment:
Is this an absolute path, relative path, or filename returned here? We will want to use relative path to root.